docs/features/theme.stories.mdx

```diff
@@ -237,9 +237,6 @@ You can only toggle the boldness of `Body`, but not `Headline` or `SubHeadline`.
 theme.grid.<BREAKPOINT>.[priority|breakpoint|cols|maxWidth|gutter]
 ```
 
-The grid provided by Circuit UI is a float-based grid. We use
-this grid in applications where we need to support older browsers, such as Android and iOS webviews.
```

Comment:
Removed this (in a couple places) because although the grid still uses float, it's not about browser support anymore. The Grid is in review: #535